activities/management/commands/mksp_jslps_geo.py

```diff
@@ -0,0 +1,149 @@
+import urllib2
```

Comment:
this is same as jslps_geo. We can just send activity in parameters and use the same code for adding geographies

Reply:
Reason already discussed with you